test.properties

```diff
@@ -3215,6 +3215,20 @@
         ) AND \
         (testray.main.component.name == "Customer Portal")
 
+    #
+    # Cookie Preferences
+    #
+
+    test.batch.class.names.includes[cookie-preferences]=\
+        **/cookies/**/*Test.java
+
+    test.batch.dist.app.servers[cookie-preferences]=tomcat
+
+    test.batch.names[cookie-preferences]=${test.batch.names[security]}
+
+    test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][cookie-preferences]=\
+        (testray.component.names ~ "Cookie Preferences")
```

Comment:
It's better to have this PQL also:
(portal.upstream == true) AND \

It's helpful to us to run specific case in the future. But it's not a mandatory requirement.